GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+# Glossary
+
+Each of these terms might refer to a conceptual class, or a particular instance. Hopefully clear in context.
+
+## `component`
+One of the main rectangles on the screen, and its corresponding definition in the `viewconf`. (If you mean React components more generally, make that explicit.)
+
+## `source`
+A URL, a `source type`, and optionally, an ID. We don't associate any semantics with the URL string. The type will determine a schema to validate against, and will be used to subtype the load `event`. The optional ID (Not yet implemented!) will allow components to be more precise in the events they publish and subscribe.
+
+## `source type`
+A string given with the source which determines the load `event` to publish. These can be broad: If a particular `component` doesn't make sense with a particular `source`, just don't use it, or specialize it by ID. Right now, I don't think we have a need for super-types or sub-types.
+
+## `layer`
+The rendering of one `source` within the spatial `component`, implemented by a Deck.gl composite layer.
```

Comment:
Yes ... except I would phrase it a little differently to avoid suggesting that we model a `has-a` relationship. Perhaps:

> A `raster` source will cause (what we now call) `LayerPublisher` (but should rename!) to broadcast `raster.add` events which are subscribed to by `Spatial` components, and linked to particular `layers`

... but that might be little too detailed, and there may be a middle ground.

Reply:
Great. I'm comfortable with this level of detail